fastchat/serve/gradio_block_arena_vision_anony.py

```diff
@@ -88,51 +89,51 @@
 
 # TODO(chris): Find battle targets that make sense
 BATTLE_TARGETS = {
-    "gpt-4-turbo": {
-        "gemini-1.5-pro-preview-0409",
-        "claude-3-opus-20240229",
-        "reka-flash-20240226",
-    },
-    "gemini-1.5-pro-preview-0409": {
-        "gpt-4-turbo",
-        "gemini-1.0-pro-vision",
-        "reka-flash-20240226",
-    },
-    "gemini-1.0-pro-vision": {
-        "gpt-4-turbo",
-        "gemini-1.5-pro-preview-0409",
-    },
-    "claude-3-opus-20240229": {
-        "gpt-4-turbo",
-        "gemini-1.5-pro-preview-0409",
-        "reka-flash-20240226",
-    },
-    "claude-3-sonnet-20240229": {
-        "claude-3-opus-20240229",
-        "gpt-4-turbo",
-        "gemini-1.0-pro-vision",
-        "gemini-1.5-pro-preview-0409",
-    },
-    "claude-3-haiku-20240307": {
-        "claude-3-opus-20240229",
-        "gpt-4-turbo",
-        "gemini-1.0-pro-vision",
-        "gemini-1.5-pro-preview-0409",
-    },
-    "llava-v1.6-34b": {
-        "gpt-4-turbo",
-        "gemini-1.5-pro-preview-0409",
-        "claude-3-opus-20240229",
-        "claude-3-sonnet-20240229",
-        "claude-3-haiku-20240307",
-    },
-    "llava-v1.6-13b": {"llava-v1.6-7b", "llava-v1.6-34b", "gemini-1.0-pro-vision"},
-    "llava-v1.6-7b": {"llava-v1.6-13b", "gemini-1.0-pro-vision"},
-    "reka-flash-20240226": {
-        "gemini-1.0-pro-vision",
-        "claude-3-haiku-20240307",
-        "claude-3-sonnet-20240229",
-    },
+    # "gpt-4-turbo": {
```

Comment:
what is this?

Reply:
The idea is that we only sample certain battle targets for each model. I think the idea behind it is that you can better calculate elo if the target is around the same level as another model. For example, it wouldn't make sense to pair up GPT-4o with Llama 2 7b - so then we have battle targets. However, in this case, we don't have a good idea of ranking yet so we don't necessarily have battle targets. So, we can't set battle targets until we test it out a bit and have a leaderboard.

Reply:
got it!